backend/siarnaq/api/episodes/models.py

```diff
@@ -314,6 +399,8 @@ class TournamentRound(models.Model):
     )
     """The tournament to which this round belongs."""
 
+    # TODO consider renaming this as "challonge_round_number"
```

Comment:
Better now than later.

Reply:
jk, we're saving breaking migrations for once this PR is as ready as can be